packages/features/auth/lib/next-auth-options.ts

```diff
@@ -391,16 +391,35 @@ export const AUTH_OPTIONS: AuthOptions = {
             name: true,
             email: true,
             role: true,
+            teams: {
+              include: {
+                team: true,
+              },
+            },
           },
         });
 
         if (!existingUser) {
           return token;
         }
 
+        // Check if the existingUser has any active teams
+        const belongsToActiveTeam =
```

Comment:
Let's abstract out the code from line 160 into a fn in this module and reuse that here.